ui/app/routes/variables.js

```diff
@@ -30,13 +30,13 @@ export default class VariablesRoute extends Route.extend(WithForbiddenState) {
         { namespace },
         { reload: true }
       );
-
       return {
         variables,
         pathTree: new PathTree(variables),
       };
     } catch (e) {
-      notifyError(this)(e);
+      notifyForbidden(this)(e);
+      return e;
```

Comment:
Passing the error state down to child (path, index) routes so they can run WithForbiddenState against it.

Reply:
This seems like a lot of extra work for capturing this state. If the error is caught on the `variables` route why can't we just edit the `variables.hbs` template to render the `ForbiddenMessage` else render the `outlet`?

Reply:
@ChaiWithJai because the ability to filter down / change your namespace exists at /index and /path, but not at the parent level. If we pass the forbidden message up here, the user has no way of escaping it. (say they're on default namespace but only have Secure Vars access on namespace-2)0

Reply:
OK, no worries.